termwiz/src/surface/line.rs

```diff
@@ -22,6 +23,12 @@ bitflags! {
         const SCANNED_IMPLICIT_HYPERLINKS = 1<<2;
         /// true if we found implicit hyperlinks in the last scan
         const HAS_IMPLICIT_HYPERLINKS = 1<<3;
+        /// The line contains 1+ cells with explicit hyperlinks set
+        const HAS_HYPERFILE = 1<<1;
```

Comment:
these bit assignments collide with those they're copying above; they should be `1<<4`, `1<<5` and so on

Reply:
Ok, I do knot know what "1<<" stands for, but I am going to make change...mmm. done.

Reply:
done